ts/features/itwallet/common/components/ItwCredentialClaim.tsx

```diff
@@ -227,6 +228,27 @@ const ImageClaimItem = ({ label, claim }: { label: string; claim: string }) => (
   />
 );
 
+/**
+ * Component which renders an attachment claim
+ * @param name - name of the file
+ */
+const AttachmentsClaimItem = ({ name }: { name: string }) => (
```

Comment:
Isn't this more of a `PdfClaimItem` rather than `AttachmentsClaimItem`? Does it make sense?

Reply:
@LazyAfternoons Yes, you are right, currently we only have PDF attachments.
But this component is intended to be the components that should be rendered for a generic attachment claim. In fact it shows a fixed label that says “Attachment”. I thought it made more sense